### PR TITLE
chore: allow running failing

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/web/src/components/expandable.tsx [hunk 1]
```diff
@@ -26,10 +26,10 @@ export const Expandable: React.FunctionComponent<React.PropsWithChildren<{
     <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', whiteSpace: 'nowrap' }}>
       <div
         className={'codicon codicon-' + (expanded ? 'chevron-down' : 'chevron-right')}
-        style={{ cursor: 'pointer', color: 'var(--vscode-foreground)', marginRight: '4px' }}
+        style={{ cursor: 'pointer', color: 'var(--vscode-foreground)', marginLeft: '5px' }}
```
Comment (dgozman) [inline packages/web/src/components/expandable.tsx:29]: Wouldn't this mess up other uses of Expandable?

File: packages/trace-viewer/src/ui/watchMode.tsx [hunk 2]
```diff
@@ -573,13 +610,27 @@ function createTree(rootSuite: Suite | undefined, projects: Map<string, boolean>
...
+  const statuses = new Set(filterTokens.filter(t => t.startsWith('s:')).map(t => t.substring(2)));
+  if (statuses.size)
+    statuses.add('running');
+  const projects = new Set(filterTokens.filter(t => t.startsWith('p:')).map(t => t.substring(2)));
+
+  const filter = (testCase: TestCaseItem) => {
+    const title = testCase.tests[0].titlePath().join(' ').toLowerCase();
+    if (!textTokens.every(token => title.includes(token)))
+      return false;
+    testCase.children = (testCase.children as TestItem[]).filter(test => !statuses.size || statuses.has(test.status));
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/watchMode.tsx:623]: nit: no need to cast anymore?

**Review by dgozman — APPROVED** [review]

File: packages/web/src/components/expandable.tsx (hunk 1 above)
Comment (pavelfeldman) [reply packages/web/src/components/expandable.tsx:29]: I'll check, it is only used for network tab...

File: packages/trace-viewer/src/ui/watchMode.tsx (hunk 2 above)
Comment (pavelfeldman) [reply packages/trace-viewer/src/ui/watchMode.tsx:623]: Still need to sadly.

**Pull request merged**